client/extensions/woocommerce/state/sites/review-replies/handlers.js

```diff
@@ -73,25 +74,27 @@ export function handleReviewRepliesRequestError( { dispatch }, action, error ) {
 }
 
 export function handleReviewReplyCreate( { dispatch }, action ) {
-	const { siteId, productId, reviewId, replyText } = action;
-
-	dispatch( request( siteId, action, '/wp/v2' ).post( 'comments', {
-		content: replyText,
-		parent: reviewId,
-		post: productId,
+	const { siteId, reviewId, replyText } = action;
+
+	// TODO - Update to use /wp/v2/comments again if possible. POST `/wp/v2/comments`
```

Comment:
Was the `/wp/v2` POST being proxied to the remote site somehow? I thought all CREATE operations in the WPCOM wp/v2 version of the REST API had been disabled.

Reply:
It's using the Jetpack server API proxy to communicate to the remote site. Check out `/developing-jetpack/jetpack-and-the-wordpress-rest-api/` on the field guide. This is how we are communicating with the WooCommerce REST API as well. As far as I can tell CREATE via this method works. This endpoint even still creates the comment, but times out before returning the correct response.